src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/ErrorHandlingTransmissionPolicy.cs

```diff
@@ -12,9 +12,10 @@
     using TaskEx = System.Threading.Tasks.Task;
 #endif
 
-    internal class ErrorHandlingTransmissionPolicy : TransmissionPolicy
+    internal class ErrorHandlingTransmissionPolicy : TransmissionPolicy, IDisposable
     {
         private BackoffLogicManager backoffLogicManager;
+        private TaskTimerInternal pauseTimer = new TaskTimerInternal { Delay = TimeSpan.FromSeconds(BackoffLogicManager.SlotDelayInSeconds) };
```

Comment:
nitpick: the delay value here is throw away because it has to be replaced with the latest value from the backoff logic manager when handling a transmission error and actually starting the timer.